src/cleanupCommand/cleanupManager.ts

```diff
@@ -106,4 +93,17 @@ export class CleanupManager {
     }
     return filteredTasks;
   }
+
+  private async deleteExpiredFailedTasks(tasks: IJob<IngestionParams>[]): Promise<IJob<IngestionParams>[]> {
+    const FAILED_CLEANUP_DELAY = this.config.get<number>('failed_cleanup_delay_days.ingestion');
```

Comment:
move "const FAILED_CLEANUP_DELAY = this.config.get<number>('failed_cleanup_delay_days.ingestion');" to ctor

Reply:
moved and done